website/addons/owncloud/__init__.py

```diff
@@ -0,0 +1,28 @@
+from . import model
```

Comment:
Easy fix. This might be from whichever addon I originally copypasta'd to make this one.